RecoEgamma/EgammaPhotonProducers/python/photons_cfi.py

```diff
@@ -80,7 +80,13 @@
     RecHitSeverityToBeExcludedEE = cleanedHybridSuperClusters.RecHitSeverityToBeExcluded,
 )
 
-photonsFromMultiCl = photons.clone(
-  photonProducer = 'photonCoreFromMultiCl'
-)
-
+photonsFromMultiCl = photons.clone()
+# Change underlying C++ type, as ugly as it could be, and add required
+# configuration parameters
```

Comment:
[a slightly less tangled definition may be made without a clone from ootPhotons, so, just a pset in the same way it's done in ootPhotons is OK] in photonsFromMultiCl_cff.py : 
```python
photonsFromMultiCl = _gedPhotons.clone(
    photonProducer = 'photonCoreFromMultiCl',
    candidateP4type = "fromEcalEnergy",
    reconstructionStep = "tmp",
    pfEgammaCandidates = "",
    valueMapPhotons = ""
    )
del photonsFromMultiCl.regressionConfig
```